### ISSUE TITLE
Return the percentage the color covers in the image

### ISSUE THREAD

**Issue by jyotiska** [fengsp/color-thief-py#1]
Hi,

I am playing with this for a last couple of days. Is it possible to return the percentage each color covers in a given image?

**Comment by richardcornish** [comment]
I'm also interested in this data. Is there another library that can do it?

**Comment by amitmadan18** [comment]
I am also interested in the same data @jyotiska @richardcornish 
Please let me know if it's possible

**Comment by richardcornish** [comment]
If @jyotiska, @amitmadan18, or anybody else needs this information, I figured it out with the help of an [issue](https://github.com/lokesh/color-thief/issues/51) at the original [color-thief repo](https://github.com/lokesh/color-thief).

Change the `palette` method at [line 356](https://github.com/fengsp/color-thief-py/blob/master/colorthief.py#L356) in colorthief.py from:

```
def palette(self):
    return self.vboxes.map(lambda x: x['color'])
```

to:

```
def palette(self):
    total = sum(self.vboxes.map(lambda x: x['vbox'].count))
    return self.vboxes.map(lambda x: x['color'] + (x['vbox'].count, total, int(x['vbox'].count / float(total) * 100)))
```

I simply added additional members to each RGB tuple:

- The pixel count of each color
- The total sum of all pixels, which will always be the same
- The integer percentage of each color

Now `palette = color_thief.get_palette(color_count=3)` will return something like `[(236, 70, 70, 1875, 1000, 18), (4, 4, 252, 625, 1000, 6), (4, 4, 4, 7500, 1000, 75)]`, giving 18%, 6%, and 75%. Tweak to your liking. Fork the repo to make your change.

**Comment by jyotiska** [comment]
Correct. If I remember correctly, I also came up with my own hack something like this to print out the histogram.

**Comment by amitmadan18** [comment]
Thanks. Even I had came up with the same solution :)

**Comment by raoulspronck** [comment]
Is is comming to the main package or do I need to make a fork myself?
